scripts/bfb-install

```diff
@@ -797,8 +812,15 @@ fi
 
 # Check NIC mode and unbind mlx5_core driver in NIC mode.
 check_nic_mode
-if [ $is_nic_mode -eq 1 -a -n "$pcie_bdf" ]; then
-  echo "$pcie_bdf" > /sys/bus/pci/drivers/mlx5_core/unbind 2>/dev/null
+if [ $is_nic_mode -eq 1 -a -n "${pcie_bd}" ]; then
+  if [[ -e /sys/bus/pci/drivers/mlx5_core/${pcie_bd}.0 ]]; then
+    echo "Unbinding ${pcie_bd}.0"
+    run_cmd_exit local "echo \"${pcie_bd}.0\" > /sys/bus/pci/drivers/mlx5_core/unbind"
```

Comment:
To be more robust, we can set a variable here, such as unbind_p0=1, and check it when doing binding.
The script should only do bind when it does unbind already.

Reply:
I mean, what it is not bind at the beginning due to some reason (such as on purpose)?  The change will try to bind it and might cause issue.